dsa-solutions/lc-solutions/2000-2099/2030-smallest-k-length-subsequence.md

```diff
@@ -0,0 +1,92 @@
+id: smallest-k-length-subsequence

```

Comment:
Add `---` on top and ensure code follows our structure https://github.com/orgs/CodeHarborHub/discussions/3369.